docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -33,6 +33,28 @@ To start a Besu node run:
 ```bash
 besu [OPTIONS] [COMMAND]
 ```
+
+### auto-log-bloom-caching-enabled
+
+```bash tab="Syntax"
+--auto-log-bloom-caching-enabled=false
```

Comment:
I feel like the syntax should be something like `--auto-log-bloom-caching-enabled=<true|false>` and the examples have one of the two values only like for instance for https://besu.hyperledger.org/en/stable/Reference/CLI/CLI-Syntax/#p2p-enabled but it seems that boolean options are not all displayed the same way in the doc, so maybe we'll make a consistency pass later.

Reply:
@NicolasMassart - agree we should fix this/make them consistent and I'm not going to do that as part of this PR. Could you raise an issue so we remember to do this?